eng/scripts/Manage-SDKSuppression-Label.ps1

```diff
@@ -0,0 +1,386 @@
+        deprecatedBreakingChange          = "CI-BreakingChange-Python-Track2"
+        deprecatedBreakingChangeApproved  = "Approved-SdkBreakingChange-Python"
+    }
+    "azure-sdk-for-trenton"          = @{
+        breakingChange                    = $null
+        breakingChangeApproved            = $null
+        breakingChangeSuppression         = $null
+        breakingChangeSuppressionApproved = $null
+        deprecatedBreakingChange          = $null
+        deprecatedBreakingChangeApproved  = $null
+    }
+}
+
+# Check if the token is available
+if ($null -eq $GithubToken) {
+    Write-Host "GitHub token is not available"
+    exit 1
+}
+
+function Get-GitHubFileContent {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$Token,
+        [Parameter(Mandatory = $true)]
+        [string]$RepoOwner,
+        [Parameter(Mandatory = $true)]
+        [string]$RepoName,
+        [Parameter(Mandatory = $true)]
+        [string]$FilePath,
+
+        [string]$Branch = "main"
+    )
+
+    if (-not $RepoOwner -or -not $RepoName -or -not $FilePath) {
+        throw "Missing required parameters: RepoOwner, RepoName, or FilePath."
+    }
+
+    $BaseUrl = "https://api.github.com/repos/$RepoOwner/$RepoName/contents/$FilePath"
+    $Url = $BaseUrl + "?ref=$Branch"
+
+    $Headers = @{
+        Authorization = "token $Token"
+        Accept        = "application/vnd.github.v3+json"
+    }
+
+    try {
+        $Response = Invoke-RestMethod -Uri $Url -Headers $Headers -Method Get
+        $DecodedContent = [System.Text.Encoding]::UTF8.GetString([Convert]::FromBase64String($Response.content))
+
+        return $DecodedContent
+    }
+    catch {
+        Write-Error "Failed to fetch file: $_"
+        return $null
+    }
+}
+
+function Get-SDKNames {
+    param (
+        [Parameter(Mandatory = $true)]
+        $BaseRepoOwner,
+        [Parameter(Mandatory = $true)]
+        $BaseRepoName,
+        [Parameter(Mandatory = $true)]
+        $BaseRepoBranch,
+        [Parameter(Mandatory = $true)]
+        $HeadRepoName,
+        [Parameter(Mandatory = $true)]
+        $HeadRepoOwner,
+        [Parameter(Mandatory = $true)]
+        $HeadRepoBranch,
+        [Parameter(Mandatory = $true)]
+        $IssueNumber,
+        [Parameter(Mandatory = $true)]
+        $SuppressionFiles,
+        [ValidateNotNullOrEmpty()]
+        [Parameter(Mandatory = $true)]
+        $AuthToken
+    )
+    $sdkNames = @()
+    foreach ($suppressionFile in $SuppressionFiles) {
+        $HeadSDKSuppressionYaml = Get-GitHubFileContent -Token $AuthToken -RepoOwner $HeadRepoOwner -RepoName $HeadRepoName -FilePath $SuppressionFile -Branch $HeadRepoBranch
+        $BaseSDKSuppressionYaml = Get-GitHubFileContent -Token $AuthToken -RepoOwner $BaseRepoOwner -RepoName $BaseRepoName -FilePath $SuppressionFile -Branch $BaseRepoBranch
+
+        if (!$HeadSDKSuppressionYaml -or !$BaseSDKSuppressionYaml) {
+            Write-Host "Unable to retrieve file content."
```

Comment:
Can you use the logger functions from the logging.ps1 under common script folder?